pennylane/drawer/style.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains styles for using matplotlib graphics.
+
+To add a new style:
+* create a private function that modifies ``plt.rcParams``.
+* add an entry to the private dictionary ``_style_map``.
+* Add an entry to ``doc/code/qml.drawer.rst``
+* Add a test in ``tests/drawer/test_style.py``
+
+Use the decorator ``_needs_mpl`` on style functions to raise appropriate
+errors if ``matplotlib`` is not installed.
```

Comment:
Should we place this information somewhere more user facing? Like in the docs somewhere ? 
I think it would be helpful to developers who might just read the docs and not look at the source code

Reply:
I don't think this information should be "user facing". If a developer  (or external contributor) is wanting to create a new style, this should be the first file they look at anyway.